core-feature-pack/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -187,22 +187,23 @@ case "$1" in
 				markerfiletimestamp=$(grep -o '[0-9]*' "$JBOSS_MARKERFILE") > /dev/null
 				if [ "$markerfiletimestamp" -gt "$currenttime" ] ; then
 					grep -i 'success:' "$JBOSS_MARKERFILE" > /dev/null
-					if [ $? -eq 0 ] ; then
+					if [ $? -eq 0 ]; then
 						launched=1
+						log_end_msg 0
+						break
+					fi
+					grep -i 'error:' "$JBOSS_MARKERFILE" > /dev/null
+					if [ $? -eq 0 ]; then
+						launched=0
+						log_end_msg 1
 						break
 					fi
 				fi
 			fi
 		done
-		if check_status; then
-			log_end_msg 0
-		else
-			log_end_msg 1
-		fi
 
 		if [ $launched -eq 0 ]; then
-			log_warning_msg "$DESC hasn't started within the timeout allowed"
-			log_warning_msg "please review file \"$JBOSS_CONSOLE_LOG\" to see the status of the service"
+			log_warning_msg "$DESC started with errors, please see server log for details."
```

Comment:
If you want to create a separate JIRA, that would be nice. I don't see a need for a separate PR, and even separate commits for the two aspects isn't critical if it's a hassle. A single commit referencing two JIRAs in its message works fine too.